junction/base/constants.py

```diff
@@ -6,8 +6,7 @@
 
 def _user_attributes(cls):
     defaults = dir(type(str('defaults'), (object,), {}))  # gives all inbuilt attrs
-    return [
-        item[0] for item in inspect.getmembers(cls) if item[0] not in defaults]
+    return [item[0] for item in inspect.getmembers(cls) if item[0] not in defaults]
```

Comment:
Why is the square bracket follow a different rule here ? Is some other rule taking over ?